source/gameengine/Ketsji/KX_Camera.cpp

```diff
@@ -56,6 +56,7 @@ KX_Camera::KX_Camera(void* sgReplicationInfo,
       m_set_projection_matrix(false),
       m_delete_node(delete_node),
 	  m_lodDistanceFactor(1.0f),
+	  m_activityCulling(false),
```

Comment:
@youle31 : There's three levels of enable option: scenes, cameras, objects. Do you agree with this ?